README.md

```diff
@@ -131,4 +131,33 @@ websearch:
   # on your system. You can specify absolute path to an image file as well,
   # e.g. `/home/me/.projektor/google.png`
   icon: web-browser
+ui:
+  # use as a string. In a near future it will be proportinal to screen size (%)
+  width: "600"
+  height: "300"
```

Comment:
Hi! Nice work :) Do I get it correctly: these values are strings to allow support of percentages in the future?

If so, could you reword the comment to something like: `These values must be quoted so they would be parsed as strings. This is required for future support of sizes proportional to screen size.`?